packages/parse5-parser-stream/test/parser-stream.test.ts

```diff
@@ -4,8 +4,22 @@ import { generateParsingTests } from 'parse5-test-utils/utils/generate-parsing-t
 import { parseChunked } from './utils/parse-chunked.js';
 import { finished } from 'parse5-test-utils/utils/common.js';
 
-generateParsingTests('ParserStream', 'ParserStream', { skipFragments: true }, (test, opts) =>
-    parseChunked(test.input, opts)
+generateParsingTests(
+    'ParserStream',
+    'ParserStream',
+    {
+        expectErrors: [
+            //NOTE: Foreign content behaviour was updated in the HTML spec.
+            //The old test suite still tests the old behaviour.
```

Comment:
It’s a bit complex to read (for me) but it seems that p5 is correct, the tests (coming from another package) are wrong, which is why they are ignored.
Should there be an issue open(ed) there? And linked here? How long between HTML changes and the tests are updated? Do we need to help/fork? If its long, how do browsers handle this? 🤔

Reply:
We run the forked test suite here, not the upstream one. This is tracked in #448. The note here is copied from https://github.com/inikulin/parse5/blob/b428b7914483a8cef21a197bb2c9bb2aceb4527a/packages/parse5/lib/parser/index.test.ts#L16-L25

Reply:
I have updated the comments to reference the issue.

Reply:
Oh right, I felt like I reviewed this already. Perhaps good to link that issue?